pyfolio/pos.py

```diff
@@ -57,13 +57,14 @@ def get_long_short_pos(positions):
 
     pos_wo_cash = positions.drop('cash', axis=1)
     longs = pos_wo_cash[pos_wo_cash > 0].sum(axis=1).fillna(0)
-    shorts = pos_wo_cash[pos_wo_cash < 0].abs().sum(axis=1).fillna(0)
+    shorts = pos_wo_cash[pos_wo_cash < 0].sum(axis=1).fillna(0)
```

Comment:
from our offline chat:

The explicit calculation would be `net_liquidation = longs - shorts + (cash + shorts)

[12:39]  
which removes the short exposure from the cash component
which then begs the simplification to just `net_liquidation = longs  + cash`
will have to think about whether that makes sense though